src/intrinsics.h

```diff
@@ -101,6 +101,7 @@
     /*  c interface */ \
     ADD_I(cglobal, 2) \
     ALIAS(llvmcall, llvmcall) \
+    ALIAS(probecall, probecall) \
```

Comment:
```suggestion
```

Reply:
I think we still need this to define `Base.probecall`?

Reply:
but we don't need to define it? It's `ccall("name", probecall)`. `Base.llvmcall !== ccall("name", llvmcall)`

Reply:
Making this a new special function form hijacking onto `ccall` seems a terrible idea 😬

Reply:
But turning it into a call ABI should be fine right? The reason is that for BpF we need to make sure the args are in the right registers

Reply:
Making it a builtin function would likely be better. Regardless, need to figure out what this does in the interpreter also

Reply:
Makes sense to me; I'll get it all properly working with `ccall` first, and then switch to a builtin.

For the interpreter, we can always dynamically lookup the probe and semaphore values from the runtime, so that should be pretty straightforward.